TOC.md

```diff
@@ -468,6 +470,7 @@
 + 版本发布历史
   + [发布版本汇总](/releases/release-notes.md)
   + v4.0
+    - [4.0.2](/releases/release-4.0.2.md)
```

Comment:
```suggestion
```
This should not be included in this PR? @uglyengineer

Reply:
yes

Reply:
But you did not commit my suggestion yet. Hmmm I'll commit it then.